Derive failing for module path for Enum

# What is the issue

This is the same as #39 but for enum's instead of structs. This exists in version `0.4.3`.

# Example 

```rust
use arbitrary;

#[derive(arbitrary::Arbitrary)]
pub enum Wrapper {
    TestA,
    TestB
}

fn main() {
    let a = Wrapper::TestA;
    match a {
        Wrapper::TestA => println!("a"),
        _ => println!("not a"),
    }
}
```

Has error:

```
error[E0433]: failed to resolve: use of undeclared type or module `Arbitrary`
 --> src/main.rs:3:10
  |
3 | #[derive(arbitrary::Arbitrary)]
  |          ^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `Arbitrary`
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `checker`.

To learn more, run the command again with --verbose.

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,6 +28,17 @@ Released YYYY-MM-DD.
 
 --------------------------------------------------------------------------------
 
+
+## 0.4.4
+
+Released 2020-04-29.
+
+### Fixed
+
+* Fixed the custom derive for enums when used via its full path (like
+  `#[derive(arbitrary::Arbitrary)]` rather than like `#[derive(Arbitrary)]`).
+
+
 ## 0.4.3
 
 Released 2020-04-28.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "arbitrary"
-version = "0.4.3" # Make sure this matches the derive crate version
-authors = ["Simonas Kazlauskas <arbitrary@kazlauskas.me>", "Brian L. Troutwine <brian@troutwine.us>", "The Rust-Fuzz Project Developers"]
+version = "0.4.4" # Make sure this matches the derive crate version
+authors = ["The Rust-Fuzz Project Developers", "Nick Fitzgerald <fitzgen@gmail.com>", "Manish Goregaokar <manishsmail@gmail.com>", "Simonas Kazlauskas <arbitrary@kazlauskas.me>", "Brian L. Troutwine <brian@troutwine.us>"]
 categories = ["development-tools::testing"]
 edition = "2018"
 keywords = ["arbitrary", "testing"]
@@ -12,7 +12,7 @@ repository = "https://github.com/nagisa/rust_arbitrary/"
 documentation = "https://docs.rs/arbitrary/"
 
 [dependencies]
-derive_arbitrary = { version = "=0.4.3", path = "./derive", optional = true }
+derive_arbitrary = { version = "=0.4.4", path = "./derive", optional = true }
 
 [dev-dependencies]
 

--- a/derive/Cargo.toml
+++ b/derive/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "derive_arbitrary"
-version = "0.4.3" # Make sure it matches the version of the arbitrary crate itself.
-authors = ["Andre Bogus <bogusandre@gmail.com>"]
+version = "0.4.4" # Make sure it matches the version of the arbitrary crate itself.
+authors = ["The Rust-Fuzz Project Developers", "Nick Fitzgerald <fitzgen@gmail.com>", "Manish Goregaokar <manishsmail@gmail.com>", "Andre Bogus <bogusandre@gmail.com>"]
 categories = ["development-tools::testing"]
 edition = "2018"
 keywords = ["arbitrary", "testing", "derive", "macro"]

--- a/derive/src/lib.rs
+++ b/derive/src/lib.rs
@@ -152,7 +152,7 @@ fn gen_size_hint_method(input: &DeriveInput) -> TokenStream {
                 #[inline]
                 fn size_hint(depth: usize) -> (usize, Option<usize>) {
                     arbitrary::size_hint::and(
-                        <u32 as Arbitrary>::size_hint(depth),
+                        <u32 as arbitrary::Arbitrary>::size_hint(depth),
                         arbitrary::size_hint::recursion_guard(depth, |depth| {
                             arbitrary::size_hint::or_all(&[ #( #variants ),* ])
                         }),

--- a/tests/path.rs
+++ b/tests/path.rs
@@ -1,0 +1,29 @@
+#![cfg(feature = "derive")]
+
+// Regression test for ensuring the derives work without Arbitrary being imported
+
+#[derive(arbitrary::Arbitrary, Clone, Debug)]
+pub struct Struct {
+    x: u8,
+    y: u8,
+}
+
+#[derive(arbitrary::Arbitrary, Clone, Debug)]
+pub struct Tuple(u8);
+
+#[derive(arbitrary::Arbitrary, Clone, Debug)]
+pub struct Unit(u8);
+
+#[derive(arbitrary::Arbitrary, Clone, Debug)]
+pub enum Enum {
+    X(u8),
+    Y(u8),
+}
+
+#[derive(arbitrary::Arbitrary, Clone, Debug)]
+struct EndingInVec(u8, bool, u32, Vec<u16>);
+
+#[derive(arbitrary::Arbitrary, Debug)]
+struct Generic<T> {
+    inner: T,
+}

